image/image.go

```diff
@@ -120,7 +120,7 @@ func unpack(w walker, dest, refName string) error {
 		return err
 	}
 
-	if err = ref.validate(w, validRefMediaTypes); err != nil {
+	if err = ref.validate(w, []string{v1.MediaTypeImageManifest}); err != nil {
```

Comment:
In the pr #51 ,Only in the validation function need to confirm the manifest and manifestlist, and I just change the unpack and create-runtime-bundel function